buildenv/docker/mkdocker.sh

```diff
@@ -582,11 +580,7 @@ if [ $version = 20.04 ] ; then
   echo "    libnftables1 \\"
 fi
   echo "    libprotobuf-c1 \\"
```

Comment:
Incidentally, we don't directly need protobuf any more, although we did in the past. Can we remove it (maybe in another PR)? There are a number of references to protobuf in this file.

Reply:
Could you create an issue for that, listing protobuf and anything else you think is now unnecessary?